libs/langchain-anthropic/src/index.ts

```diff
@@ -1 +1,2 @@
 export * from "./chat_models.js";
+export * from "./output_parsers.js";
```

Comment:
This change is biggest question for me - if it's inline with overall "strategy" that I can export this from Anthropic and use it in community? Any thoughts?
I only need single function from there, so that could be a minor copy-paste I assume.

Reply:
Yeah I think that's fine. Maybe we can prefix it with an underscore to show that it shouldn't be imported directly.